packages/flame-types/src/types.ts

```diff
@@ -351,6 +351,10 @@ export class EvmCurrency {
   }
 }
 
+export interface ErrorWithMessage {
```

Comment:
Why did you create this? What was wrong with
```
error instanceof Error && error.message.includes("User rejected")
```

The built in `Error` type has a `message` property and this is the pattern we already use elsewhere in this codebase.

Reply:
My bad I have not seen it and did not know that was a thing